app/javascript/components/Results/FedRegister/FedRegister.tsx

```diff
@@ -42,7 +42,7 @@ const getFedRegisterAgenciesText = (agencyNames: string[]) => {
 
 const getFedRegDocPageInfo = (document: FedRegisterDoc) => {
   let content = `Pages ${document.startPage} - ${document.endPage} `;
-  content += `(${document.pageLength} page) `;
+  content += document.pageLength === 1 ? `(${document.pageLength} page) ` : `(${document.pageLength} pages) `;
```

Comment:
Hi @stevenbarragan I don't think so. I am implementing code at. https://github.com/yogesh27/search-gov/blob/main/app/helpers/federal_register_documents_helper.rb#L38 in javascript since there is no `pluralize` method in javascript

Reply:
`pluralize` in Rails does use localization. We should use  i18n for this https://www.npmjs.com/package/i18n-js#pluralization